src/app/views/users/UsersListController.jsx

```diff
@@ -0,0 +1,183 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { push, replace } from "react-router-redux";
+import PropTypes from "prop-types";
+
+import users from "../../utilities/api-clients/user";
+import notifications from "../../utilities/notifications";
+import log from "../../utilities/logging/log";
+import auth from "../../utilities/auth";
+
+import { addAllUsers } from "../../config/actions";
+import SimpleSelectableList from "../../components/simple-selectable-list/SimpleSelectableList";
+import BackButton from "../../components/button/BackButton";
+import Input from "../../components/Input";
+
+const propTypes = {
+    rootPath: PropTypes.string.isRequired,
+    dispatch: PropTypes.func.isRequired,
+    children: PropTypes.element,
+    params: PropTypes.shape({
+        userID: PropTypes.string,
+    }).isRequired,
+    loggedInUser: PropTypes.object,
+    users: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+};
+
+export class UsersListController extends Component {
```

Comment:
Hi Nick, I can see it is in draft still, but just to give you heads up. It would be great if we start avoiding classes where possible and use functional components instead. Also, this file is no way any controller just UI, can you please name it `UsersList`. Please also make use of store.

Reply:
Thanks Anna, I'm addressing these comment.